src/helpers/generateSizeString.js

```diff
@@ -0,0 +1,10 @@
+export function generateSizeString(tv) {
+    const screenSizes = tv.availableSizes.map(size => `${size} inch (${Math.round(size * 2.54)} cm)`);
+
+    if (screenSizes.length === 1) {
+        return screenSizes[0];
+    } else {
+        return screenSizes.join(' | ');
+    }
```

Comment:
Ook als het maar één size heeft dan kun je altijd nog de .join aanroepen, omdat die dan eigenlijk niet zoveel doet. Snap dat het dan geen zin heeft, maar het scheelt wel code 😉 
```suggestion
    return screenSizes.join(' | ');
```